src/lib/alarmMgr/AlarmManager.h

```diff
@@ -38,26 +40,44 @@ class AlarmManager
 {
 private:
   long long                   badInputs;
+  long long                   badInputResets;
   long long                   notificationErrors;
+  long long                   notificationErrorResets;
   long long                   dbErrors;
+  long long                   dbErrorResets;
   bool                        dbOk;
   std::map<std::string, int>  notificationV;
   std::map<std::string, int>  badInputV;
   int                         notificationErrorLogSampling;
   int                         badInputLogSampling;
+  sem_t                       sem;
 
 public:
   AlarmManager();
   AlarmManager(int _notificationErrorLogSampling, int _badInputLogSampling);
 
+  int  init(void);
+
+  int  semInit(void);
```

Comment:
sem methods could probably go in the "private sector" :-)
init() is used outside.   I'll test moving the sem methods